workspace/lib/lita/handlers/teamcity.rb

```diff
@@ -616,6 +651,17 @@ def format_artifacts_build(build_type, build_id, build_number)
         return "<#{artifacts_build_url}|#{build_type} (#{build_number})>"
       end
 
+      def format_result_successful_build(build)
+        return "\n<#{build['webUrl']}|#{build['buildTypeId']}> (ID: #{build['number']})"
+      end
+
+      def get_teamcity_since_date()
+        current_time_utc = Time.now.utc
+        current_time_teamcity = current_time_utc.in_time_zone('Pacific Time (US & Canada)')
+        start_of_previous_day_teamcity = (current_time_teamcity - 24*60*60).strftime("%Y%m%dT000000%z")
```

Comment:
Please make `24*60*60` into a variable like `seconds_to_days` so its clear what it is